packages/calcite-components/src/components/input-date-picker/input-date-picker.scss

```diff
@@ -1,55 +1,82 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+
+ * @prop --calcite-input-date-picker-background-color: Specifies the component's background color.
+ * @prop --calcite-input-date-picker-border-color: Specifies the component's border color.
+ * @prop --calcite-input-date-picker-shadow: Specifies the component's shadow.
+
+ * @prop --calcite-input-date-picker-divider-background-color: Specifies the component's divider background color.
+ * @prop --calcite-input-date-picker-icon-color: Specifies the component's icon color.
+ * @prop --calcite-input-date-picker-icon-color-hover: Specifies the component's icon color on hover.
+
+ * @prop --calcite-input-date-picker-input-background-color: Specifies the component's input background color.
+ * @prop --calcite-input-date-picker-input-border-color: Specifies the component's input border color.
+ * @prop --calcite-input-date-picker-input-corner-radius: Specifies the component's input corner radius.
+ * @prop --calcite-input-date-picker-input-shadow: Specifies the component's input shadow.
+
+ * @prop --calcite-input-date-picker-input-icon-color: Specifies the component's input icon color.
+ * @prop --calcite-input-date-picker-input-text-color: Specifies the component's input text color.
+ * @prop --calcite-input-date-picker-input-placeholder-text-color: Specifies the component's input placeholder text color.
+ */
+
 :host {
   @extend %component-spacing;
   @include floating-ui-elem-anim(".menu-container");
   @apply relative
     inline-block
     w-full
     overflow-visible
-    align-top
-    shadow-none;
+    align-top;
+
+  box-shadow: var(--calcite-input-date-picker-shadow, --calcite-shadow-none);
```

Comment:
Does this also need to be applied to the `date-picker` or does it need its own CSS prop? cc @macandcheese